Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender.py

```diff
@@ -1007,12 +1044,39 @@ def handle_incident_close_out_or_reactivation(delta: dict, incident_status: Inci
             # this functionality awaits https://jira-dc.paloaltonetworks.com/browse/CRTX-151123?filter=-2
             # once resolved the microsoft365classification field needs to be returned to the close form in order to get
             # the classification and determination fields in delta.
-            if delta.get("closeReason") == "FalsePositive" and delta.get("classification") != "FalsePositive":
-                delta.update({"classification": "FalsePositive", "determination": "Other"})
-                demisto.debug("Microsoft Defender 365 - Updating classification and determination to FalsePositive and Other")
-            elif delta.get("closeReason") == "Other" or delta.get("closeReason") == "Duplicate":
-                delta.update({"classification": "Unknown", "determination": "NotAvailable"})
-                demisto.debug("Microsoft Defender 365 - Updating classification and determination to Unknown and NotAvailable")
+
```

Comment:
please use `argToBoolean(param)` as the `custom_xsoar_to_defender_close_reason` may be 'false' which considered as true in Python